code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -449,42 +449,55 @@
 
 /obj/item/device/dogborg/sleeper/compactor/afterattack(var/obj/item/target, mob/living/silicon/user, proximity)//GARBO NOMS
 	hound = loc
-	if(!proximity)
-		return
-	if(target.anchored)
-		return
-	if(length(contents) > 32)
-		user << "<span class='warning'>Your [src.name] is full. Eject or process contents to continue.</span>"
+	if(isobj(target))
+		if(length(contents) > 11)//grow that tum after a certain junk amount
+			hound.sleeper_r = 1
```

Comment:
The icons for the bellies are identical without the lights so it's not a noticeable change. A person inside gives the bulge regardless of the clutter amount and since the ejection is all at once it shouldn't have appearance problems which might appear if having person and just a few items being left inside was even possible.

Anyway the other things could be tidied up a bit yeah.